tests/test_model.py

```diff
@@ -0,0 +1,186 @@
+from unittest import mock
+
+import pytest
+
+from context import reporchestrator
+
+
+def test_choose_in():
+    low, high = 1, 3
+    chosen = reporchestrator.choose_in(low, high)
+    assert low <= chosen <= high
+
+
+def test_periodic_commit():
+    def split():
+        return 42
+    commit = reporchestrator.periodic_commit(1, 2, split)
+    assert commit == 85
+
+
+def test_next_member():
+    c = {"developers": ["a", "b"], "developer_strategy": "round-robin"}
+    gen = reporchestrator.next_member(c)
+    m = next(gen)
+    assert m == "a"
+    m = next(gen)
+    assert m == "b"
+    m = next(gen)
+    assert m == "a"
+
+
+def test_next_member_random_uniform():
+    c = {"developers": ["a", "b"], "developer_strategy": "random-uniform"}
+    gen = reporchestrator.next_member(c)
+    m = next(gen)
+    assert m in ("a", "b")
```

Comment:
It is not easy to test these distributions. If I understand correctly, here the ```m``` can be generated by any distributions, the test will pass. Anyway, I think it is a good start for testing it, so no need to change anything. 👍